views/components/collapsibleBlock.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * All About Olaf
+ * Individual row for inside a collapsible view
+ */
+
+import React from 'react'
+import {
+  View,
+  TouchableOpacity,
+} from 'react-native'
+
+import Collapsible from 'react-native-collapsible'
+
+
+export default class CollapsibleBlock extends React.Component {
+
+  static propTypes = {
+    borderColor: React.PropTypes.string.isRequired,
+    children: React.PropTypes.array.isRequired,
+  }
+
+  state = {
+    isCollapsed: true,
+  }
+
+  style() {
```

Comment:
The advantage to `StyleSheet.create` is that RN can process the stylesheet once, then reference a cache to load the styles in the future, rather than needing to build the entire style tree every time. Could you extract `this.style` into an external `styles` variable created with `StyleSheet.create`, then reference it in the `render` method, please?

That will require that you use the array form of `style={[styles.collapsedContent, {backgroundColor: this.props.borderColor}]}` for the `collapsedContent` block.